dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -254,10 +250,7 @@ void Microsoft::Windows::AppNotifications::Helpers::RegisterAssets(std::wstring
 
     if (displayName.length() == 0)
     {
-        wil::unique_cotaskmem_string processName;
-        THROW_IF_FAILED(GetCurrentProcessPath(processName));
-
-        displayName = processName.get();
+        THROW_IF_FAILED(wil::GetModuleFileNameExW(GetCurrentProcess(), nullptr, displayName));
```

Comment:
@danielayala94 Should we guarantee that the displayname here starts with uppercase?

Reply:
For example, it should be Tiktok. Not tiktok.

Reply:
This returns the original path name. So it will be uppercase (original-casing).

Reply:
While for the demo app this is going to retrieve a good-looking display name, we should consider this as a fix while working on introducing the default icon.

Reply:
I have created a work item for it -> #2228